test/lib/buffer-utils.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { Buffer } from "buffer";
+import * as crypto from "crypto";
+import { bufferToSha1 } from "../../lib/buffer-utils";
+
+describe("buffer-utils", () => {
+  describe("bufferToSha1", () => {
```

Comment:
I guess this test i related with the previous commit, right? Do you think it will be better merge both commits in one in this case?